sw/device/lib/dif/dif_aes.h

```diff
@@ -361,6 +361,27 @@ dif_result_t dif_aes_load_data(const dif_aes_t *aes, const dif_aes_data_t data);
 OT_WARN_UNUSED_RESULT
 dif_result_t dif_aes_read_output(const dif_aes_t *aes, dif_aes_data_t *data);
 
+/**
+ * Process a stream of data containing the plain text and and outbut a stream of
+ * data with the cipher text.
+ *
+ * This function should be used when performance is desired. It requires the
+ * automatic operation mode activated.
```

Comment:
Would it make sense to check that automatic operation is indeed activated? This could be done as the control register is indeed readable. But it would require to add a function for that. Anyway, I am not sure it's really needed.

Reply:
I don't think it is necessary, as the automatic operation is part of the `dif_aes_transaction_t` in the `dif_aes_start` which is mandatory before this function.

Reply:
Ah, okay. Thanks for the clarification. This sounds good.